ocular/src/config.rs

```diff
@@ -2,27 +2,22 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct ChainClientConfig {
-    pub key: String,
     pub chain_id: String,
     #[serde(rename = "rpc-addr")]
     pub rpc_address: String,
     #[serde(rename = "grpc-addr")]
     pub grpc_address: String,
     pub account_prefix: String,
-    pub keyring_backend: String,
     pub gas_adjustment: f64,
     pub gas_prices: String,
-    pub key_directory: String,
```

Comment:
The path where the keys are stored should be developer configurable at some level. This would support weird stuff that people might want to do and we can hide it behind a default for most applications